wftests/warrior_tests/suites/framework_tests/ts_framework_tests.xml

```diff
@@ -22,13 +22,13 @@
                         <onError action="next"/>
                         <impact>impact</impact>
                 </Testcase>
-                <Testcase>
+        <!--       <Testcase>
                         <path>../../testcases/framework_tests/tc_runmode_checks_on_rup_ruf_rmt_with_intervals.xml</path>
                         <context>positive</context>
                         <runtype>sequential_keywords</runtype>
                         <onError action="next"/>
                         <impact>impact</impact>
-                </Testcase>
+                </Testcase> -->
```

Comment:
remove this testcase if not applicable/available

Reply:
Removed the testcase.